### PR TITLE
Remove ExecuteTrajectory method

### PR DESCRIPTION
**Pull request by felixvd**
As discussed, execution of raw trajectories is dangerous so it should not be a public API.

cc @kanbouchou

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
+<<<<<<< HEAD
```
Comment (kanbouchou) [inline CHANGELOG.md:1]: looks like merge error?

Comment (felixvd) [reply CHANGELOG.md]: Fixed. Forgot to save after resolving the conflict.

**Comment by kanbouchou** [comment]
Thanks @felixvd looks good to me. 
@rdiankov can you just double check? I'm fine with deleting executetrajectory for my projects.

**Comment by rdiankov** [comment]
thanks

**Pull request merged**